android/gltfio-android/libgltfio-jni.symbols

```diff
@@ -0,0 +1 @@
+_Java_com_google_android_filament_*
```

Comment:
Is this file being referenced anywhere? I realize that `libfilamat` has one as well, but I can't seem to find where it gets used. I might have committed it accidentally.

Reply:
It should be referenced in the CMake file as a linker argument.

Reply:
Done.